vagrant.md

```diff
@@ -41,29 +41,6 @@ will ssh into your Minerva VM.
 
 will re-provision your Minerva VM with ansible.
 
-#### vagrant provision, now with even more tags
-
-If you already have a provisioned VM, you can set tags through environment variables
-to reprovision a limited number of steps.  You can set the tags as a comma separated
-list like
-
-    export MINERVA_VAGRANT_ANSIBLE_TAGS='minerva-update,test'
-
-##### minerva
-
-Think of this tag as minerva with dependencies.
-
-This tag will pull the git version of minerva on your VM that is your current git revision on your host,
-install system dependencies and pip dependencies defined in the minerva role,
-then install the minerva plugin to girder and build the client side code.
-
-##### minerva-update
```

Comment:
@kotfic 

Here's a description of minerva-update.  Is it still valid or should it be removed?